onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -189,7 +191,8 @@ std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_CPU(in
 std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_CUDA(OrtDevice::DeviceId device_id,
                                                                                OrtCudnnConvAlgoSearch cudnn_conv_algo_search,
                                                                                size_t cuda_mem_limit,
-                                                                               onnxruntime::ArenaExtendStrategy arena_extend_strategy);
+                                                                               onnxruntime::ArenaExtendStrategy arena_extend_strategy,
+                                                                               bool do_copy_in_compute_stream);
```

Comment:
>do_copy_in_compute_stream [](start = 84, length = 25)

change to do_copy_in_default_stream to make it consistent

Reply:
done